scripts/xtensa-build-all.sh

```diff
@@ -411,6 +416,12 @@ do
 		cp "$OVERRIDE_CONFIG" override.config
 	fi
 
+	if [[ "$IPC_VERSION" == "IPC4" ]]
+	then
+	    IPC4_CONFIG_OVERLAY_FILE="${SOF_TOP}/src/arch/xtensa/configs/override/$IPC4_CONFIG_OVERLAY.config"
+	    cp "$IPC4_CONFIG_OVERLAY_FILE" override.config
```

Comment:
```suggestion
	    cat "${SOF_TOP}/src/arch/xtensa/configs/override/$IPC4_CONFIG_OVERLAY.config \
	    >> override.config
```
Creating a new variable gives the wrong impression that it's going to be used somewhere else. Moreover the name is very similar to the other variable which is another potential source of confusion.

Keep this next to the other `override.config` options, having the menuconfig in the middle of the `override.config` does not seem to make sense.

Reply:
@marc-hb  thanks for the comments! I've updated the commit.
It's more direct and intuitive now :) no risk of variable confusion

Reply:
> Keep this (new lines) close to the (big block of) other override.config options

Please?

Reply:
I did not notice it because the menuconfig looks in the middle of override.config operations originally.
Now I moved the code of this commit （append something to override.config）after menuconfig change, while menuconfig  change is still after the copy of the whole override.config file, is it OK ？

Reply:
> I did not notice it because the menuconfig looks in the middle of override.config operations originally.

Yes and this is not ideal but the first bit of override.config code that is before menuconfig is quite different from the other ones if you look carefully.